src/components/AddDiveForm/AddDive.js

```diff
@@ -1,14 +1,16 @@
 import styled from 'styled-components';
 import { useState } from 'react';
-
-import ButtonBack from '../Button/Button';
+import { IoMdArrowRoundBack } from 'react-icons/io';
+import Button from '../Button/Button';
 
 export default function AddDive({ handleTogglePage, onCreateDive }) {
   const [diveData, setDiveData] = useState('');
 
   return (
     <>
-      <ButtonBack name="back" onClick={handleTogglePage} />
+      <ButtonBack variant="back" name="back" onClick={handleTogglePage}>
```

Comment:
If you create a `ButtonBack` with the word `back` in it, you could directly set the `variant` while defining that styled component. Use the `.attrs` method for that.

Reply:
`handleTogglePage` does not sound like a good function name, if you call it from a back button. Maybe `onClickBack`? Or just `onBack`?